test/images/image-util.sh

```diff
@@ -182,7 +183,8 @@ build() {
 
     docker buildx build --progress=plain --no-cache --pull --output=type="${output_type}" --platform "${os_name}/${arch}" \
         --build-arg BASEIMAGE="${base_image}" --build-arg REGISTRY="${REGISTRY}" --build-arg OS_VERSION="${os_version}" \
-        -t "${REGISTRY}/${image}:${TAG}-${suffix}" -f "${dockerfile_name}" .
+        -t "${REGISTRY}/${image}:${TAG}-${suffix}" -f "${dockerfile_name}" \
+	--label "image_version=${TAG}" --label "commit_id=${GIT_COMMIT_ID}" .
```

Comment:
Wondering if we can add git_url as well which saves the location..

Reply:
Done